ocaml/xapi/xapi_pgpu.ml

```diff
@@ -422,6 +426,7 @@ let nvidia_vf_setup ~__context ~pf ~enable =
       raise (fail msg)
     | Some 0 when Sys.file_exists script ->
       debug "PCI %s has 0 VFs - calling %s" pci script;
+      dequarantine pci;
```

Comment:
I think this is redundant, because this is always called after `bind_to_nvidia`, but since this is an idempotent operation it is good practice to do it whenever we "assert" that the device needs to be in Dom0.

Reply:
It still isn't redundant if the device was already bound to the nvidia driver, because `bind_to_nvidia` does not call `dequarantine` then. However, it may be simpler to just always dequarantine in `bind_to_nvidia` (as the first thing that is done in that function), and not here anymore, so that there is just a single point where the `dequarantine` function is called.